mlir/include/mlir/Dialect/Linalg/TransformOps/LinalgTransformOps.td

```diff
@@ -2162,7 +2166,16 @@ def VectorizeOp : Op<Transform_Dialect, "structured.vectorize",
 
   let results = (outs);
 
-  let hasCustomAssemblyFormat = 1;
+  let assemblyFormat = [{
+    $target oilist(
```

Comment:
Same as above.

Reply:
i did remove for this as i believe the intention is that `tile_sizes` is never optional, nor does it make sense to allow empty list